code/modules/clothing/under/accessories/storage.dm

```diff
@@ -67,6 +67,31 @@
 	icon_state = "vest_white"
 	slots = 5
 
+/obj/item/clothing/accessory/storage/overalls
+	name = "overalls"
+	desc = "Heavy-duty overalls for use on the work site, with plenty of convenient pockets to boot."
+	icon_state = "mining_overalls"
+	overlay_state = "mining_overalls"
+
+/obj/item/clothing/accessory/storage/overalls/mining
+	name = "shaft miner's overalls"
+	desc = "Heavy-duty overalls. Ostensibly for your protection, not vacuum-rated. Comes with convenient pockets for miscellaneous tools."
+	slots = 5
```

Comment:
Instead of having all of these have 5 slots, just make the base type have 5 slots